packages/note/index.d.ts

```diff
@@ -0,0 +1,303 @@
+type Note = string;
+type Midi = number;
+type OrNull<T> = T | null;
+type NoteLetter = "A" | "B" | "C" | "D" | "E" | "F" | "G";
+type NoteAccidental = "#" | "b";
+type Octave = number;
+
+type NoteProps = {
+  name: Note;
+  letter: NoteLetter;
+  acc: NoteAccidental; // {String}: the note accidentals
```

Comment:
This is not exactly true, because `##` or `###` can be the accidentals (not just one...)